x-pack/legacy/plugins/lens/public/types.ts

```diff
@@ -158,7 +158,8 @@ export interface OperationMetadata {
   // A bucketed operation is grouped by duplicate values, otherwise each row is
   // treated as unique
   isBucketed: boolean;
-
+  isHistogram?: boolean;
+  scale?: 'ordinal' | 'interval' | 'ratio';
```

Comment:
I'm not sure I understand the need for these additions- can you explain?

Reply:
The histogram information is used to enable histogram mode if that makes sense for the operation. The scale is not necessary for the currently supported operations, but will come in handy later. E.g. if we do a terms of a number field, the data type will be number, but we shouldn't use a linear scale in that case. This field makes it possible for the visualization to react to that differences (e.g. providing different suggestions). See also the PR description.